### ISSUE TITLE
fleetctl start X -> doesn't start

### ISSUE THREAD

**Issue by haf** [coreos/fleet#1202]
I'm having trouble using fleet to start a service that starts just fine on two of the other nodes in the cluster. You can read the background: https://github.com/deis/deis/issues/3580

I do a `fleetctl start deis-store-metadata.service` and it returns 0 - that's good. `fleetctl list-units` show all other units are up and running, but on this single machine.

Then if I go to that machine:

``` bash
# systemctl start deis-store-metadata
Failed to start deis-store-metadata.service: Unit deis-store-metadata.service failed to load: No such file or directory
```

Logs from fleet.service:

```
Apr 26 22:21:51 deis-3 fleetd[624]: INFO client.go:291: Failed getting response from http://localhost:4001/: cancelled
Apr 27 13:23:03 deis-3 fleetd[624]: INFO manager.go:145: Triggered systemd unit deis-store-metadata.service stop: job=28411665
Apr 27 13:23:03 deis-3 fleetd[624]: INFO reconcile.go:321: AgentReconciler completed task: type=StopUnit job=deis-store-metadata.service reason="unit currently launched but desired state is loaded"
Apr 27 13:23:38 deis-3 fleetd[624]: ERROR manager.go:136: Failed to trigger systemd unit deis-store-metadata.service start: Unit deis-store-metadata.service failed to load: No such file or directory.
Apr 27 13:23:38 deis-3 fleetd[624]: INFO reconcile.go:321: AgentReconciler completed task: type=StartUnit job=deis-store-metadata.service reason="unit currently loaded but desired state is launched"
Apr 27 13:25:27 deis-3 fleetd[624]: INFO manager.go:145: Triggered systemd unit deis-store-metadata.service stop: job=28411836
Apr 27 13:25:27 deis-3 fleetd[624]: INFO reconcile.go:321: AgentReconciler completed task: type=StopUnit job=deis-store-metadata.service reason="unit currently launched but desired state is loaded"
Apr 27 13:25:45 deis-3 fleetd[624]: ERROR manager.go:136: Failed to trigger systemd unit deis-store-metadata.service start: Unit deis-store-metadata.service failed to load: No such file or directory.
Apr 27 13:25:45 deis-3 fleetd[624]: INFO reconcile.go:321: AgentReconciler completed task: type=StartUnit job=deis-store-metadata.service reason="unit currently loaded but desired state is launched"
```

**Comment by scole-scea** [comment]
Might be #1160. See the helper service at https://gist.github.com/scole-scea/177a367c51d0b93d51f6. This was supposed to be fixed by fleet 0.9.2, and I'm guessing your hosts are running 0.9.1.

**Comment by haf** [comment]
v0.9.0 actually, Deis v1.4.1. We're planning on upgrading to v1.5 - is it fixed there? I'll verify after upgrade and close this issue.

**Comment by scole-scea** [comment]
I can't answer your question about Deis: it's not something I'm familiar with. The `systemd-reload.service` in that Gist I posted is a service designed to work around the issue, if you're distribution doesn't have a more current fleet. (We use that service for our cluster, and it's completely handled the issue.)

Look at the other linked issues for more comments about what's really going on, to see if what you're seeing is actually the same issue.

**Comment by bcwaldon** [comment]
This was addressed by v0.9.2
